polynote-frontend/polynote/messages.js

```diff
@@ -600,10 +600,27 @@ export class KernelBusyState extends KernelStatusUpdate {
 
 KernelBusyState.codec = combined(bool, bool).to(KernelBusyState);
 
+export class KernelInfo extends KernelStatusUpdate {
+    static get msgTypeId() { return 3; }
+
+    static unapply(inst) {
+        return [inst.content];
+    }
+
+    constructor(content) {
```

Comment:
Can we make this hold a dictionary of name/value pairs instead of a single string?

Reply:
Sure, will do!